packages/wonder-blocks-dropdown/components/__tests__/search-text-input.test.js

```diff
@@ -0,0 +1,123 @@
+// @flow
+import * as React from "react";
+import {shallow} from "enzyme";
+
+import SearchTextInput from "../search-text-input.js";
```

Comment:
naming suggestion: `SearchField`.  You using this terminology elsewhere in this PR, but beyond that I think it generalizes better, e.g.
- `TextField`
- `PasswordField`
- `SearchField`

Reply:
I agree, but I'd prefer to keep it that way to avoid modifying the functionality we already have in place for `MultiSelect`. I wanted to touch as less as I could here to avoid doing any unnecessary refactors on the original implementation. As you mentioned I'll add a new ticket/TODO to work on this in the future.